Source/Core/Core/IOS/ES/Identity.cpp

```diff
@@ -114,6 +118,96 @@ IPCCommandResult ES::Sign(const IOCtlVRequest& request)
   m_ios.GetIOSC().Sign(sig_out, ap_cert_out, m_title_context.tmd.GetTitleId(), data, data_size);
   return GetDefaultReply(IPC_SUCCESS);
 }
+
+ReturnCode ES::VerifySign(const std::vector<u8>& hash, const std::vector<u8>& ecc_signature,
+                          const std::vector<u8>& certs_bytes)
+{
+  if (!SConfig::GetInstance().m_enable_signature_checks)
+    return IPC_SUCCESS;
```

Comment:
Should we log a message here, in case someone disables signature checks unintentionally?